### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by emymaria**
For supporting the invocation of the Pattern Atlas UI by the QC Atlas UI the Pattern Atlas UI component is added to the docker-compose

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -72,6 +72,21 @@ services:
       - '1977:1977'
     networks:
       - default
+  pattern-atlas-ui:
+    image: patternpedia/patternrepo-ui:latest
```
Comment (salmma) [inline docker-compose.yml]: should be patternatlas/...
shouldn't it @manuwei?

File: docker-compose.yml [hunk 2]
```diff
@@ -72,6 +72,21 @@ services:
...
+    environment:
+      # because the requests are performed from the browser and not
+      # from the container itself we have to specify localhost here
+      # and **not** the container name
+      PATTERN_ATLAS_API_HOST_NAME: localhost
+      PATTERN_ATLAS_API_PORT: 1977
+      LATEX_RENDERER_HOST_NAME: localhost
+      LATEX_RENDERER_PORT: 5030
+    ports:
+      - "4200:80"
```
Comment (salmma) [inline docker-compose.yml]: outgoing port should be changed to 81, so I guess 81:80.

File: docker-compose.yml (hunk 1 above)
Comment (salmma) [inline docker-compose.yml]: change from latest to the last version, I guess v1.2.0

**Review by salmma — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (emymaria) [reply docker-compose.yml]: I got error when I changed patterpedia to patternatlas.
ERROR: pull access denied for patternatlas/patternrepo-ui, repository does not exist or may require 'docker login': denied: requested access to the resource is denied

Comment (emymaria) [reply docker-compose.yml]: Done the last 2 changes

Comment (salmma) [reply docker-compose.yml]: the [PR](https://github.com/PatternAtlas/pattern-atlas-docker/pull/5/files) in PatternAtlas is now merged and you can try it again :)

File: docker-compose.yml [hunk 3]
```diff
@@ -72,6 +72,21 @@ services:
...
+    environment:
+      # because the requests are performed from the browser and not
+      # from the container itself we have to specify localhost here
+      # and **not** the container name
+      PATTERN_ATLAS_API_HOST_NAME: localhost
+      PATTERN_ATLAS_API_PORT: 1977
+      LATEX_RENDERER_HOST_NAME: localhost
+      LATEX_RENDERER_PORT: 5030
+    ports:
+      - "81:80"
```
Comment (salmma) [inline docker-compose.yml]: sorry for giving you the wrong port number! Please change to 1978:80 :)

**Review by salmma — CHANGES_REQUESTED** [review]
Port number change and try again from patternpedia to patternatlas :)

File: docker-compose.yml (hunk 3 above)
Comment (emymaria) [reply docker-compose.yml]: done

File: docker-compose.yml (hunk 1 above)
Comment (emymaria) [reply docker-compose.yml]: done

**Review by lharzenetter — APPROVED** [review]

**Comment by salmma** [comment]
branch will be merged as soon as new versions for qc-atlas-ui, pattern-atlas-api, and pattern-atlas-ui are set

File: docker-compose.yml [hunk 4]
```diff
@@ -59,7 +59,7 @@ services:
     networks:
       - default
   pattern-atlas-api:
-    image: patternatlas/pattern-atlas-api:v1.4.1
+    image: patternatlas/pattern-atlas-api:v1.5.1
```
Comment (lharzenetter) [inline docker-compose.yml:66]: What happend to 1.5.0?

**Review by lharzenetter — APPROVED** [review]

**Review by salmma — APPROVED** [review]
LGTM

**Pull request merged**